azure-servicebus/tests/test_servicebus_servicebus.py

```diff
@@ -509,6 +510,8 @@ def test_send_queue_message_with_custom_message_properties(self):
         self.assertEqual(received_msg.custom_properties['floating'], 3.14)
         self.assertEqual(
             received_msg.custom_properties['dob'], datetime(2011, 12, 14))
+        self.assertEqual(
+            received_msg.custom_properties['quote_message'], 'This "should" work fine')
```

Comment:
Good point, I don't know if single quote is something special in C#, I will add a test. Thanks!